ext/drsyms/CMakeLists.txt

```diff
@@ -218,25 +219,25 @@ if (WIN32)
   append_property_string(TARGET drsyms_static COMPILE_FLAGS "/DSTATIC_LIB")
 endif (WIN32)
 DR_install(FILES "${dwarf_libpath}" "${elf_libpath}" "${elftc_libpath}"
-  DESTINATION ${INSTALL_EXT_LIB})
-if (WIN32)
- DR_install(FILES "${dwarf_dir}/dwarf.pdb" "${dwarf_dir}/elftc.pdb"
-    DESTINATION ${INSTALL_EXT_LIB}
-    PERMISSIONS ${owner_access} OWNER_EXECUTE GROUP_READ GROUP_EXECUTE
-    WORLD_READ WORLD_EXECUTE)
-endif (WIN32)
+        DESTINATION ${INSTALL_EXT_LIB})
+if (WIN32 AND MSVC)
```

Comment:
See above where we will probably use the MSVC-built libelftc libraries.  Even if MinGW does not handle .pdb files, we are exporting these libraries for use by clients, and clients may well use MSVC and want the .pdb files -- even if none of the DR libs have .pdb files, I suppose.

Reply:
@derekbruening See comment https://github.com/DynamoRIO/dynamorio/pull/3852#discussion_r329346054 That is same.